test/reporters/html/utility_functions_test.dart

```diff
@@ -1,6 +1,8 @@
+import 'package:dart_code_metrics/src/models/file_report.dart';
 @TestOn('vm')
```

Comment:
@dkrutskikh seems that the annotation should be first

Reply:
fixed